engineio/asyncio_client.py

```diff
@@ -247,10 +250,11 @@ def _reset(self):
             headers = headers.copy()
             headers['Cookie'] = cookies
 
+        sslopt = {'certfile': cert}
         try:
             ws = await websockets.connect(
                 websocket_url + self._get_url_timestamp(),
-                extra_headers=headers)
+                extra_headers=headers, sslopt=sslopt)
```

Comment:
Are you sure this is right? My understanding was that this websocket client takes an `SSLContext` object in the `ssl` argument, I don't believe `sslopt` is a valid argument.